Resources/public/ts/StingerConfiguration.ts

```diff
@@ -1,6 +1,7 @@
 export class StingerConfiguration {
     attr: any
     ajaxUrl?: any
+    ajaxData?: object
```

Comment:
This also doesn't sound like "data" is given. Should be renamed to something more meaningful